.markdownlint-cli2.jsonc

```
@@ -189,6 +189,13 @@
           "search": "](https://developer.mozilla.org/",
           "replace": "](/",
           "skipCode": true
+        },
+        {
+          "name": "bad-spelling",
```

nit:
```suggestion
          "name": "incorrect-spelling",
```

Not sure if the value of the `name` property is defined by the library, but if not, I would recommend renaming it to something a little less bad 😄

---

No rule has been hard coded in [the library](https://github.com/OnkarRuikar/markdownlint-rule-search-replace/#configuration).

We can create more rules like this:
```
{
  "default": true,
  "search-replace": {
    "rules": [
      {
        "name": "product-names",
        "message": "Use correct cases for trademarks and registered names",
        "search": ["OS x", "Javascript", "Github", "Typescript"],
        "replace": ["macOS", "JavaScript", "GitHub", "TypeScript"],
        "skipCode": false
      }
    ]
  }
}
```